build/webpack.base.config.js

```diff
@@ -71,26 +66,12 @@ module.exports = {
     logLevel,
     getBaseConfig(config) {
         return {
-            mode: environment,
-            devtool: isProd ? 'source-map' : 'eval-source-map',
+            mode: nodeEnv,
             output: {
                 publicPath: '/dist/',
                 filename: isProd ? '[name].[chunkhash].js' : '[name].js',
                 chunkFilename: isProd ? '_[name].[chunkhash].js' : '_[name].js',
             },
-            resolve: {
-                alias: {
-                    '@components': path.resolve(config.rootDir, 'src/components'),
-                    '@dist': path.resolve(config.rootDir, 'dist'),
-                    '@js': path.resolve(config.rootDir, 'src/js'),
-                    '@scss': path.resolve(config.rootDir, 'src/scss'),
-                    '@server': path.resolve(config.rootDir, 'src/server'),
-                    '@src': path.resolve(config.rootDir, 'src'),
-                    '@static': path.resolve(config.rootDir, 'static'),
-                    '@store': path.resolve(config.rootDir, 'src/store'),
-                },
-                extensions: ['*', '.js', '.vue', '.json'],
-            },
```

Comment:
Breaking change - no longer providing anything out of the box here - this configuration is left up to the consuming application build